team.js

```diff
@@ -19,6 +19,7 @@ const members = {
         He is interested in Cryptography, Security, Consensus Protocols, Decentralised Finance, and Ethereum.
       </Box>
     ),
+    image: "team/shresth_agrawal.jpg",
```

Comment:
Let's move this file to a separate folder, `/data/team.js` maybe. Also, does it need to be JS or can it be JSON?